Tests/Hashing/HashingTest.php

```diff
@@ -26,16 +27,27 @@ public function testBasicArgon2iHashing()
         if (!defined('PASSWORD_ARGON2I')) {
             $this->markTestSkipped('PHP not compiled with Argon2i hashing support.');
         }
+        $hashing = new ArgonHashing(['memory' => 512, 'time' => 5, 'threads' => 3]);
+        $value = $hashing->make('password');
+        $this->assertNotSame('password', $value);
+        $this->assertTrue($hashing->verify('password', $value));
+        $this->assertFalse($hashing->needsRehash($value));
+        $this->assertFalse($hashing->needsRehash($value, ['memory' => 512, 'time' => 5, 'threads' => 3]));
```

Comment:
This assertion is failed when check whether the hash needs hashing again.

Reply:
false

Reply:
https://www.php.net/manual/en/function.password-needs-rehash.php

Reply:
The assertions has been fixed.